app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreen.kt

```diff
@@ -85,25 +88,25 @@ class EnterOtpScreen(context: Context, attributeSet: AttributeSet) : RelativeLay
     userPhoneNumberTextView.text = phoneNumberWithCountryCode
   }
 
-  fun goBack() {
+  override fun goBack() {
```

Comment:
Since we are no longer using `goBack` in the controller, should we have no extracted it in the UI interface?

Reply:
Good question, let me check and see if we can remove this.

Reply:
I checked and it's still used in the controller. We only moved handling of back clicks, but the method itself is still invoked from the controller if the login OTP arrives and the user gets logged in in the background.